plutono/README.md

```diff
@@ -581,8 +539,6 @@ stringData:
   client_secret: <value>
 ```
 
-Include in the `extraSecretMounts` configuration flag:
```

Comment:
this got lost in migration

Reply:
good catch. fixed in 876cbf470b01da81d0d3070efb4fbfefec1acb07